test/hotspot/jtreg/gc/TestSystemGC.java

```diff
@@ -24,39 +24,46 @@
 package gc;
 
 /*
- * @test TestSystemGCSerial
+ * @test id=serial
```

Comment:
In recent changes we used upper-case for the id labels. I think it would be good to be consistent and do the same here.